config.yaml

```diff
@@ -4,9 +4,9 @@ data:
   categories: categories.yaml
 schemas:
   apps: src/static/schemas/v2/apps.schema.json
```

Comment:
I am not sure it is a good idea to take files from the different versions. Can we take all from one instead?

Reply:
I thought about this a bit and those are findings. Pros for maintaining all schema files in one version:

- Configuration file is simplified
 
Cons:
- Full copy of all schema files required for each backwards-incompatible change in individual files.
- It actually creates more confusion since we are giving the impression that schema files must all be of the same version to be compatible, which is simply not the case (new files can rely on files from previous versions). Case in point, your comments on this PR.
- Slightly harder to guarantee backwards-compatibility since it requires a manual check that the schema files have in fact not changed.*

*) The point of this PR is to restore compatibility with the v1 schema, this is obviously the case since I am using that schema for validation.

Maybe this would have been easier/clearer if we had versioned individual files right from the start, i.e., `apps.v1.schema.json` instead of `v1/apps.schema.json`. We could still make that change, but it would of course be a bit awkward. I checked and GitHub pages does not support HTTP redirects which makes such a change even more difficult.

As a final option, we could just change `apps.schema.json` without creating a new versions since we are likely the only ones using it, but we would still be breaking an API contract.

Reply:
Per offline discussion, we will adopt the approach of maintaining all schemas for one version in the same tree.